### ISSUE TITLE
Missing softmax for rpn scores

### ISSUE THREAD

**Issue by FortiLeiZhang** [chenyuntc/simple-faster-rcnn-pytorch#67]
In RegionProposalNetwork
```python
rpn_scores = self.score(h)
rpn_scores = rpn_scores.permute(0, 2, 3, 1).contiguous()
rpn_fg_scores = rpn_scores.view(n, hh, ww, n_anchor, 2)[:, :, :, :, 1].contiguous()
rpn_fg_scores = rpn_fg_scores.view(n, -1)
rpn_scores = rpn_scores.view(n, -1, 2)
```
The score is raw score, not the probability that a proposal being foreground or background. For a single proposal, the sum of two numbers should be equal to 1 (because they are binary classification), however, when you print them out, they are small numbers like [-0.0742, -0.0999].

And in the following ProposalCreator:
```python
        # Sort all (proposal, score) pairs by score from highest to lowest.
        # Take top pre_nms_topN (e.g. 6000).
        order = score.ravel().argsort()[::-1]
```
These scores are used to rank the proposal, which is not correct. Because the comparison of raw score numbers between different proposals are meaningless, and will yield wrong ranking.
